ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/banners.j2

```diff
@@ -1,13 +1,13 @@
-{# banners #}
-{% if banners is defined and banners is not none %}
-{%     if banners.login is defined and banners.login is not none %}
+{# eos - banners #}
+{% if banners is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` does a deep test, this `if banners is arista.avd.defined` should be removed. Or move `!` just under this test.